components/inputs/input-text.js

```diff
@@ -298,7 +306,7 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 	}
 
 	render() {
-		const isFocusedOrHovered = !this.disabled && (this._focused || this._hovered);
+		const isFocusedOrHovered = !this.disabled && (this._focusedOrHovered);
```

Comment:
The re-render only really cares if they're focused OR hovered -- it doesn't matter if they hover and then move away if they're still focused.